catalog/app/website/pages/AwsMarketplace/Features.tsx

```diff
@@ -0,0 +1,87 @@
+import cx from 'classnames'
+import * as React from 'react'
+import * as M from '@material-ui/core'
+
+import dataCommons from './dataCommons.svg'
+import dataVersionControl from './dataVersionControl.svg'
+import visualizations from './visualizations.svg'
+
+const useFeatureStyles = M.makeStyles((t) => ({
+  root: {
```

Comment:
Padding on this is good, but the behavior on shrink is odd -- my feeling is we should go from 3 across the 3 stacked for mobile. 

@akarve @robnewman @fiskus what are your thoughts on this? Effort to achieve? Certainly not a top-priority, but worth a call out.

Reply:
> from 3 across the 3 stacked for mobile
yes and done